gradle.properties

```diff
@@ -1,28 +1,33 @@
 # Project-wide Gradle settings.
+
 # IDE (e.g. Android Studio) users:
 # Gradle settings configured through the IDE *will override*
 # any settings specified in this file.
+#
 # For more details on how to configure your build environment visit
 # http://www.gradle.org/docs/current/userguide/build_environment.html
+
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xms1024m -Xmx4096m -Dfile.encoding=UTF-8
+
+# Enable Gradle configuration and build cache to speed up builds on developer machines.
+org.gradle.configuration-cache=true
```

Comment:
This enables the Gradle configuration cache, which caches the result of Gradle's configuration phase between builds if nothing has changed. This results in a speed-up between builds.